examples/vector_databases/azuresearch/Getting_started_with_azure_ai_search_and_openai.ipynb

```diff
@@ -556,49 +640,59 @@
       "\n",
-      "Title: Battle of Bannockburn\n",
-      "URL: https://simple.wikipedia.org/wiki/Battle%20of%20Bannockburn\n",
-      "Caption: The Battle of Bannockburn, fought on 23 and 24 June 1314, was an important<em> Scottish</em> victory in the Wars of<em> Scottish</em> Independence. A smaller Scottish army defeated a much larger and better armed English army. Background  When King Alexander III of Scotland died in 1286, his heir was his granddaughter Margaret, Maid of Norway.\n",
+      "Title: Printing\n",
+      "Reranker Score: 1.603400707244873\n",
+      "URL: https://simple.wikipedia.org/wiki/Printing\n",
+      "Caption: Machines to speed printing, cheaper paper, automatic stitching and binding all arrived in the 19th century during the industrial revolution. What had once been done by a few men by hand was now done by limited companies on huge machines. The result was much lower prices, and a much wider readership.\n",
       "\n",
-      "Title: First War of Scottish Independence\n",
-      "URL: https://simple.wikipedia.org/wiki/First%20War%20of%20Scottish%20Independence\n",
-      "Caption: The First War of<em> Scottish Independence</em> lasted from the outbreak of the war in 1296 until the 1328. The Scots were defeated at Dunbar on 27 April 1296. John Balliol abdicated in Montrose castle on 10 July 1296.\n",
+      "Title: Industrialisation\n",
+      "Reranker Score: 1.3238357305526733\n",
+      "URL: https://simple.wikipedia.org/wiki/Industrialisation\n",
+      "Caption: <em>Industrialisation</em> (or<em> industrialization)</em> is a process that happens in countries when they start to use machines to do work that was once done by people.<em> Industrialisation changes</em> the things people do.<em> Industrialisation</em> caused towns to grow larger. Many people left farming to take higher paid jobs in factories in towns.\n",
       "\n"
      ]
     }
    ],
    "source": [
     "# Semantic Hybrid Search\n",
-    "query = \"Famous battles in Scottish history\" \n",
+    "query = \"What were the key technological advancements during the Industrial Revolution?\"\n",
     "\n",
-    "search_client = SearchClient(search_service_endpoint, index_name, AzureKeyCredential(search_service_api_key))  \n",
-    "vector = Vector(value=generate_embeddings(query), k=3, fields=\"content_vector\")  \n",
+    "search_client = SearchClient(search_service_endpoint, index_name, credential)\n",
+    "vector_query = VectorizedQuery(\n",
+    "    vector=generate_embeddings(query, deployment),\n",
+    "    k_nearest_neighbors=3,\n",
+    "    fields=\"content_vector\",\n",
+    ")\n",
     "\n",
-    "results = search_client.search(  \n",
-    "    search_text=query,  \n",
-    "    vectors=[vector], \n",
+    "results = search_client.search(\n",
```

Comment:
Is query language no longer needed in semantic ranker queries in this API version?

Reply:
Correct, in azure-search-documents==11.4.0 it defaults to the 2023-11-01 Stable/GA API Version which removes `queryLanguage` as a query parameter.